utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -0,0 +1,37 @@
+package org.oppia.util.data;
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.widget.TextView
+
+const val CUSTOM_TAG = "oppia-noninteractive-image"
```

Comment:
These variables could be a bit clearer in terms of what values they correspond to. Consider:
- ``CUSTOM_IMG_TAG``
- ``REPLACE_IMG_TAG``
- ``CUSTOM_IMG_FILE_PATH_ATTRIBUTE``
- ``REPLACE_IMG_FILE_PATH_ATTRIBUTE``

Reply:
Changed these variable to their context.